cli/cmd/vuln_container_list_assessments.go

```diff
@@ -370,13 +372,16 @@ func (v *treeCtrVuln) ParseData(data []api.VulnerabilityContainer) {
 	for _, ctr := range data {
 		latestContainer, exist := v.Get(ctr.ImageID)
 		if exist {
-			if latestContainer.EvalGUID == ctr.EvalGUID {
+			if latestContainer.EvalGUID == ctr.EvalGUID && ctr.StartTime.Before(latestContainer.StartTime) {
```

Comment:
why do we need this extra evaluation? it feels redundant 🤔

Reply:
It's not redundant with the current order of the code. We don't want the same evalID to fall into the continue if it's a later start time. I could restructure this function, but I'd rather make minimal changes for this fix.